test/test18.py

```diff
@@ -0,0 +1,19 @@
+import unittest
+
+def missing_num(arr):

```

Comment:
Modified the test after almost an hour of searching for a way to do it. Phew! It was done finally. The solution will work on Linux perfectly. For Windows, a bit of modification might be required.
Thanks a lot for appreciating. It always feels good to know that my contribution is helping you :)